### PR TITLE
Nested stacks with button issues

### PR DESCRIPTION
**Pull request by ericstraily**
The timeout buttons are not responding correctly anymore

### CODE REVIEW

File: uwh-refbox/src/main.rs [hunk 1]
```diff
@@ -482,104 +481,544 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
...
+        advantage_type_select_layout.attach(&advantage_equal_select_button, 1, 1, 1, 2);
+        advantage_type_select_layout.attach(&advantage_b_select_button, 2, 1, 1, 2);
+
+        advantage_add_layout.attach(&advantage_keypad, 0, 0, 4, 9);
+        advantage_add_layout.attach(&advantage_type_select_layout, 4, 0, 8, 3);
+        advantage_add_layout.attach(&penalty_selection_grid, 4, 3, 8, 6);
+
+
+        // Setting up the white/equal/black selected buttons
+        advantage_w_select_button.connect_clicked(clone!(@strong advantage_equal_select_button, @strong advantage_b_select_button => move |b| {
```
Comment (TristanDebrunner) [inline uwh-refbox/src/main.rs:689]: These buttons and the "type of foul" buttons need an initial value. For each set, should choose one and `.set_active(true)` it right after the `.connect_clicked()`s

File: uwh-refbox/src/main.rs [hunk 2]
```diff
@@ -482,104 +481,544 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
...
+                && !penalty_delay_game_button.get_active()
+                && !penalty_illegal_adv_button.get_active()
+                && !penalty_illegal_stop_button.get_active()
+                && !penalty_out_bounds_button.get_active()
+                && !penalty_illegal_sub_button.get_active()
+                && !penalty_grab_wall_button.get_active()
+            {
+                b.set_active(true);
+            }
+        }));
```
[the remark this answers is not in the document]
Comment (TristanDebrunner) [reply uwh-refbox/src/main.rs:749]: See #45

File: uwh-refbox/src/main.rs [hunk 3]
```diff
@@ -334,69 +344,61 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
...
+        w_score_minus_button.set_margin_top(BUTTON_MARGIN);
+        w_score_minus_button.set_margin_bottom(BUTTON_MARGIN);
+        let b_score_plus_button = new_button("+", &["blue-modifier"]);
+        b_score_plus_button.set_margin_end(BUTTON_MARGIN);
+        b_score_plus_button.set_margin_top(BUTTON_MARGIN);
+        b_score_plus_button.set_margin_bottom(BUTTON_MARGIN);
+        let b_score_minus_button = new_button("-", &["blue-modifier"]);
+        b_score_minus_button.set_margin_end(BUTTON_MARGIN);
+        b_score_minus_button.set_margin_top(BUTTON_MARGIN);
+        b_score_minus_button.set_margin_bottom(BUTTON_MARGIN);
```
[the remark this answers is not in the document]
Comment (ericstraily) [reply uwh-refbox/src/main.rs:362]: I wasn't able to get this to work, not sure if I am doing something wrong or if they have to be individually called out like what I have

File: uwh-refbox/src/main.rs [hunk 4]
```diff
@@ -334,69 +344,61 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
...
+        let score_edit_submit_button = new_button("SUBMIT", &["green"]);
+
+        let edit_score_page_header = new_label("MANUALLY ADJUST SCORES", "header-gray");
         let white_score_header = new_label("WHITE SCORE", "header-white");
         let black_score_header = new_label("BLACK SCORE", "header-black");
         let modified_white_score = new_label("#W", "modified-score-white");
         let modified_black_score = new_label("#B", "modified-score-black");
         let empty_score_edit_label = gtk::Label::new(None);
 
         let white_score_header_box = gtk::Grid::new();
```
[the remark this answers is not in the document]
Comment (ericstraily) [reply uwh-refbox/src/main.rs:374]: adding the set spacing doesn't seem to do what I wantchanged 'box' to 'grid'

[the remark this answers is not in the document]
Comment (ericstraily) [reply uwh-refbox/src/main.rs:374]: changed 'box' to 'grid'

File: uwh-refbox/src/main.rs [hunk 5]
```diff
@@ -482,104 +481,544 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
...
+        let penalty_b_select_button = new_toggle_button("BLACK", &["black"]);
+
+        let penalty_1min_top_label = new_label("1", "penalty-top-yellow");
+        let penalty_1min_bottom_label = new_label("MINUTE", "penalty-bottom-yellow");
+        let penalty_2min_top_label = new_label("2", "penalty-top-orange");
+        let penalty_2min_bottom_label = new_label("MINUTE", "penalty-bottom-orange");
+        let penalty_5min_top_label = new_label("5", "penalty-top-red");
+        let penalty_5min_bottom_label = new_label("MINUTE", "penalty-bottom-red");
+        let penalty_dismiss_top_label = new_label("X", "penalty-top-blue");
+        let penalty_dismiss_bottom_label = new_label("DISMISS", "penalty-bottom-blue");
```
[the remark this answers is not in the document]
Comment (ericstraily) [reply uwh-refbox/src/main.rs:494]: hmm, will check this out

File: uwh-refbox/src/main.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (ericstraily) [reply uwh-refbox/src/main.rs:749]: I thought so too, I have a vague idea how it might work but no idea how to get started on it

**Comment by TristanDebrunner** [comment]
Deprecated by #56

**Pull request closed without merging**